CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.89.0
+
+### Enhancements
+
+* RealmObjectSchema.isPrimaryKey(String) (#2440)
+
+### Bug fixes
+
+* RealmObjectSchema.isRequired and RealmObjectSchema.isNullable don't throw when the give field names don't exist.
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  jcjacksonengineering@gmail.com
    Domain joe-jackson.net has exceeded the max emails per hour (104/100 (104%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext4.iad.github.net ([192.30.252.195]:60933 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by rs5.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.86_1)
	(envelope-from <noreply@github.com>)
	id 1aflgI-001eIP-1j
	for jcjackson@joe-jackson.net; Tue, 15 Mar 2016 05:54:26 -0400
Date: Mon, 14 Mar 2016 09:16:28 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1457972188;
	bh=mTQsuSqeyI1ADA7y9OgotveHtTESE7NKVyHI2YJJ5ac=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=gb9Xf9QXAgDBBrksVgL1wCBXc9Nsw5zVrhen7CCY/Nlccn6PlRBuI862HLdq6D2oi
	 9nTw0XAAuBe8t/505vQKkivRlNhFBb34OMxvZk+hGjdr6ZmD9IutM+51xNBeZ/pGPX
	 y2HoDWnRgges0uCAGky0IPaCUbyuOKCCnfUPGCXY=
From: Christian Melchior <notifications@github.com>
Reply-To: realm/realm-java <reply@reply.github.com>
To: realm/realm-java <realm-java@noreply.github.com>
Message-ID: <realm/realm-java/pull/2445/r56027019@github.com>
In-Reply-To: <realm/realm-java/pull/2445@github.com>
References: <realm/realm-java/pull/2445@github.com>
Subject: Re: [realm-java] isPrimaryKey/getPrimaryKey on RealmObjectSchema
 (#2445)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_56e6e3dc91f82_3d093fc5a537f2bc10548eb";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: cmelchior
X-GitHub-Recipient: jcj52436999
List-ID: realm/realm-java <realm-java.realm.github.com>
List-Archive: https://github.com/realm/realm-java
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+000f94f79264f7bd309a10fcc4c4dd09c74cac83e4601ce492cf0000000112fea5dc92a170ce0356e78b@reply.github.com>,
 <https://github.com/notifications/unsubscribe/AA-U955evO2tewDe7p24tIRNCJuHQBakks5ptYncgaJpZM4Hv5lo>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: jcjackson@joe-jackson.net


----==_mimepart_56e6e3dc91f82_3d093fc5a537f2bc10548eb
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

> @@ -1,3 +1,13 @@
> +## 0.89.0
> +
> +### Enhancements
> +
> +* RealmObjectSchema.isPrimaryKey(String) (#2440)
> +
> +### Bug fixes
> +
> +* RealmObjectSchema.isRequired and RealmObjectSchema.isNullable don't throw when the give field names don't exist.

Use `()` after method names. It makes it a lot more readable IMO

---
Reply to this email directly or view it on GitHub:
https://github.com/realm/realm-java/pull/2445/files#r56027019
----==_mimepart_56e6e3dc91f82_3d093fc5a537f2bc10548eb
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/realm/realm-java/pull/2445#discussion_r56027019">CHANGELOG.md</a>:</p>
<pre style='color:#555'>&gt; @@ -1,3 +1,13 @@
&gt; +## 0.89.0
&gt; +
&gt; +### Enhancements
&gt; +
&gt; +* RealmObjectSchema.isPrimaryKey(String) (#2440)
&gt; +
&gt; +### Bug fixes
&gt; +
&gt; +* RealmObjectSchema.isRequired and RealmObjectSchema.isNullable don&#39;t throw when the give field names don&#39;t exist.
</pre>
<p>Use <code>()</code> after method names. It makes it a lot more readable IMO</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/realm/realm-java/pull/2445/files#r56027019">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/AA-U95JmrEFeuq9iWWvc81YEY35G8ECoks5ptYncgaJpZM4Hv5lo.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
<div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
  <link itemprop="url" href="https://github.com/realm/realm-java/pull/2445/files#r56027019"></link>
  <meta itemprop="name" content="View Pull Request"></meta>
</div>
<meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_56e6e3dc91f82_3d093fc5a537f2bc10548eb--